openapi/ga4gh-tool-discovery.yaml

```diff
@@ -505,19 +507,21 @@ definitions:
             - 630d31c3-381e-488d-b639-ce5d047a0142
             - 'dockstore.org:630d31c3-381e-488d-b639-ce5d047a0142'
             - 'bio.tools:630d31c3-381e-488d-b639-ce5d047a0142'
+            - 'https://bio.tools/bioexcel_seqqc'
```

Comment:
discussing this with @denis-yuen this should be in `contains` and not in `aliases` because multiple TRS entries can point to the same bio.tools entry (different versions, different environments, etc.)

Reply:
:+1: